lib/SiftClient.php

```diff
@@ -544,7 +544,7 @@ public function getDecisions($opts = array()) {
         }
 
         try {
-            $request = new SiftRequest($url, SiftRequest::GET, $opts['timeout'], self::API3_VERSION, array('auth' => $this->api_key . ':'));
+            $request = new SiftRequest($url, SiftRequest::GET, $opts['timeout'], self::API3_VERSION, array('auth' => $this->api_key . ':', 'params' => $params));
```

Comment:
@sabitha-varghese could we add some test cases covering this?

Reply:
@viaskal-sift  Reviewed and looks good to merge.